code/datums/jobs/job/deathsquad.dm

```diff
@@ -187,7 +187,7 @@
 	mask = /obj/item/clothing/mask/gas/PMC
 	head = /obj/item/clothing/head/helmet/marine/veteran/PMC/commando
 	glasses = /obj/item/clothing/glasses/night/m56_goggles
-	suit_store = /obj/item/weapon/gun/rifle/standard_smartmachinegun/ds
+	suit_store = /obj/item/weapon/gun/rifle/standard_smartmachinegun
```

Comment:
unrelated change

Reply:
/obj/item/weapon/gun/rifle/standard_smartmachinegun/ds is deleted